pageserver/src/tenant/timeline.rs

```diff
@@ -309,6 +310,11 @@ pub struct Timeline {
     /// Configuration: how often should the partitioning be recalculated.
     repartition_threshold: u64,
 
+    /// Configuration: how often should we check if new image layers should be created
+    image_layer_creation_check_threshold: ArcSwap<u64>,
```

Comment:
Other than this, it looks good. Wouldn't this have to be a global like an `Arc<ArcSwap<_>>` for this to work as a PageserverConf value? I think an `Arc<AtomicU64>` would work just as well.

Otherwise I think this is looking good. With gc feedback removed we cannot get layer creation pressure from there. I wonder if this will make us create less image layers though and allow less gc at the cost of saving a lot CPU cycles... Well, it'd be a win anyways.

Reply:
The threshold became a tenant config in the end.